### PR TITLE
[IMP] purchase_operating_unit: Enable Multi Picking/Warehouse on PO's

### PR DESCRIPTION
**Pull request by osi-scampbell**
Do not throw the UserError if we have a warehouse that does not have an operating unit (shared warehouse)

### CODE REVIEW

File: purchase_operating_unit/models/purchase_order.py [hunk 1]
```diff
@@ -92,6 +92,10 @@ def _onchange_operating_unit_id(self):
             types = type_obj.search([('code', '=', 'incoming'),
                                      ('warehouse_id.operating_unit_id', '=',
                                       self.operating_unit_id.id)])
+            if not types:
+                types = type_obj.search([('code', '=', 'incoming'),
+                                     ('warehouse_id.operating_unit_id', '=',
+                                      False)])
```
Comment (max3903) [inline purchase_operating_unit/models/purchase_order.py]: @osi-scampbell what about using:
```
types = type_obj.search([
    ('code', '=', 'incoming'),
    ('warehouse_id.operating_unit_id', 'in', (self.operating_unit_id.id, False)
])
```

Comment (max3903) [reply purchase_operating_unit/models/purchase_order.py]: @osi-scampbell I meant to merge this search with the one before (line 92)...

**Review by max3903 — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by bodedra — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
It is ok to me as soon lint checks are fixed.

**Pull request merged**